packages/postcss-colormin/test/index.js

```diff
@@ -235,14 +238,62 @@ test(
 test(
   'should convert long color to 8-digit hex when supported',
   processCSS('h1{color:rgba(100% 50% 0% / 50%)}', 'h1{color:#ff800080}', {
-    env: 'chrome62',
+    overrideBrowserslist: 'Chrome 62',
   })
 );
 
 test(
   'should convert long color to 4-digit hex when supported',
   processCSS('h1{color:hsla(0 100% 50% / 40%)}', 'h1{color:#f006}', {
-    env: 'chrome62',
```

Comment:
So `env` did not change meaning, it's just that `chrome62` is not an environment anymore because of the changes to the Browserslist configuration  in the tests, right?

Reply:
I've used this option to add test coverage for going straight to Browserslist. It skips both the **.browserslist** config and **package.json** environments

Using `from` (files) or `path` (directories) will locate the nearest config though